cli/lib/kontena/scripts/completer

```diff
@@ -3,238 +3,7 @@
 
 # resolve bin path, ignoring symlinks
 require 'pathname'
-require 'yaml'
 bin_file = Pathname.new(__FILE__).realpath
+$:.unshift File.expand_path('../../..', bin_file)
```

Comment:
Ah yes, I see now. The fixed version expands to `cli/lib/kontena/scripts/completer/../../..` = `cli/lib`.